generators/node/obmConfig.json

```diff
@@ -0,0 +1,55 @@
+{
+    "host": "localhost",
+    "port": 4001,
+    "prefix": "bvd-receiver",
+    "protocol": "http",
+    "apiKey": "081508150815081508150815",
+    "generators": [
+      {
+        "name":"Health Status",
+        "sample": {
+          "mdb": ["OBM Health Status"],
+          "label": ["Overall"],
+          "type": ["PIE"],       
+          "numberOfCritical" : {"range": [0,10], "format": "int"},
+          "numberOfMajor" : {"range": [0,10], "format": "int"},
+          "numberOfMinor" : {"range": [0,10], "format": "int"},
+          "numberOfNormal" : {"range": [0,10], "format": "int"},
+          "numberOfUnknown" : {"range": [0,10], "format": "int"},
+          "mostCritical" : "(function (data) { if (data.numberOfCritical >0 ) {return 'CRITICAL';} else if (data.numberOfMajor >0 ) {return 'MAJOR';} else if (data.numberOfMinor > 0) {return 'MINOR';} else return 'NORMAL';})"
+        },
+        "frequency": 500, 
+        "tolerance": 500,
+        "tags":"omi,mdb",
+        "dims":"mdb,label,type"
+      },
+      {
+        "name":"KPI Status",
+        "sample": {
+          "viewName": ["OprSample"],
+          "ciName": ["IT JBoss Cluster"],
+          "kpiName": ["System Performance"],
+          "status": ["Critical","Major","Minor","Warning", "Info","Ok","Down Time","No Data","UnKnown"]
+        },
+        "frequency": 500, 
+        "tolerance": 500,
+        "tags":"omi,kpi",
+        "dims":"viewName,ciName,kpiName"
+      },
+      {
+        "name":"KPI Status Employee service",
```

Comment:
If you have two KPI sample generators, that only differ in the values for ciName and kpiName, then you can merge them and make both parameters an array of the two values.
Like
ciName: [ IT JBoss Cluster, Employee Self Service]